src/config/section/user.js

```diff
@@ -69,6 +69,14 @@ export default {
       dataView: true,
       show: (record) => { return record.state === 'enabled' }
     },
+    {
+      api: 'authorizeSamlSso',
+      icon: 'check-circle',
+      label: 'Configure SAML SSO Authorization',
+      dataView: true,
+      popup: true,
+      component: () => import('@/views/iam/ConfigureSamlSsoAuth.vue')
```

Comment:
Could an AutogenView be used instead with mapping for enabling & disabling based on a flag?

Reply:
I think, yes it can be refactored to have 2 action buttons - one for enabling and one for disabling based on the state - can remove the ConfigureSamlAuth component altogether